### PR TITLE
pythonPackages.googlemaps: init at 4.4.2

### PR DESCRIPTION
**Pull request by Scriptkiddi**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/googlemaps/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, requests
+, responses, pytestCheckHook, pytest, pytestcov
+}:
+
+buildPythonPackage rec {
+  pname = "googlemaps";
+  version = "4.4.2";
+
```
Comment (IvarWithoutBones) [inline pkgs/development/python-modules/googlemaps/default.nix:9]: Build fails on python2: 
```
  DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support
in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
  Processing ./googlemaps-4.4.2-py2-none-any.whl
  ERROR: Package 'googlemaps' requires a different Python: 2.7.18 not in '>=3.5'
cannot build derivation '/nix/store/wpfc7hqvj0mcfx6qzfys0l4pn0bjkh7b-env.drv': 1 dependencies couldn't be built
```

```suggestion
  disabled = isPy27;
```

**Review by IvarWithoutBones — CHANGES_REQUESTED** [review]
Result of `nixpkgs-review pr 100027` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python27Packages.googlemaps</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python37Packages.googlemaps</li>
    <li>python38Packages.googlemaps</li>
  </ul>
</details>

File: pkgs/development/python-modules/googlemaps/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
```
Comment (IvarWithoutBones) [inline pkgs/development/python-modules/googlemaps/default.nix:1]: `isPy27` is not included by default.
```suggestion
{ stdenv, buildPythonPackage, fetchFromGitHub, isPy27
```

Comment (Scriptkiddi) [reply pkgs/development/python-modules/googlemaps/default.nix:1]: thanks i missed that

**Pull request merged**